File: namsor/typescript-node/src/ReligionIndianFullBatchExample.ts
import * as fs from 'fs';
import api from "namsor_client"
import models from "namsor_client"

const apiCaller = new api.IndianApi();
apiCaller.setApiKey(api.IndianApiApiKeys.api_key, "YOUR_API_KEY");

const personalNames1 = new models.PersonalNameSubdivisionIn();
personalNames1.id = "e630dda5-13b3-42c5-8f1d-648aa8a21c42";
personalNames1.name = "Akash Sharma";
personalNames1.subdivisionIso = "IN-PB";

const personalNames = [
  personalNames1,
];

const batchPersonalNameSubdivisionIn = new models.BatchPersonalNameSubdivisionIn();
batchPersonalNameSubdivisionIn.personalNames = personalNames;

apiCaller.religionIndianFullBatch(
  batchPersonalNameSubdivisionIn,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling IndianApi#religionIndianFullBatch:");
  console.log(error.body);
});

File: namsor/kotlin/src/main/kotlin/oseg/namsor_examples/ReligionIndianFullBatchExample.kt
package oseg.namsor_examples

import app.namsor.client.infrastructure.*
import app.namsor.client.apis.*
import app.namsor.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map

class ReligionIndianFullBatchExample
{
    fun religionIndianFullBatch()
    {
        ApiClient.apiKey["api_key"] = "YOUR_API_KEY"

        val personalNames1 = PersonalNameSubdivisionIn(
            id = "e630dda5-13b3-42c5-8f1d-648aa8a21c42",
            name = "Akash Sharma",
            subdivisionIso = "IN-PB",
        )

        val personalNames = arrayListOf<PersonalNameSubdivisionIn>(
            personalNames1,
        )

        val batchPersonalNameSubdivisionIn = BatchPersonalNameSubdivisionIn(
            personalNames = personalNames,
        )

        try
        {
            val response = IndianApi().religionIndianFullBatch(
                batchPersonalNameSubdivisionIn = batchPersonalNameSubdivisionIn,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling IndianApi#religionIndianFullBatch")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling IndianApi#religionIndianFullBatch")
            e.printStackTrace()
        }
    }
}
